.github/workflows/integration.yml

```diff
@@ -246,15 +246,14 @@ jobs:
         # Produces ghcr.io/agoric/agoric-sdk:latest used in the following upgrade test.
         # run: cd packages/deployment && ./scripts/test-docker-build.sh | $TEST_COLLECT
         # XXX skip TAP test output and collection for now; it hides the output from the logs
```

Comment:
Is this comment obsolete?

Reply:
Nope. That script is a wrapper for `test-docker-build` which is still part of CI for building the SDK image. This PR doesn't have anything do with that.